docs/content/guides/operator/snapshots.mdx

```diff
@@ -54,9 +88,49 @@ When you restore a Full node from a snapshot, write it to the path `/opt/sui/db/
 
 :::
 
-## S3 buckets used per environment
+### Restoring using Formal snapshots
+
+To restore using a Formal snapshot, use the `sui-tool` binary. `sui-tool` can be downloaded along with other sui binaries.
+See [Install Sui](../developer/getting-started/sui-install.mdx) for more details.
+
+The following steps can be used to restore a node from a formal snapshot:
+
+1. If it's running, stop the node.
+2. Run the command:
+   ```
+   sui-tool download-db-snapshot --epoch "<EPOCH-NUMBER>" --genesis "<PATH-TO-GENESIS-BLOB>" \
```

Comment:
Sorry to do this, but after our debugging with @tharbert it seems we will need to add `--network` in, or specify the archival bucket and bucket type as well (if they pass in `--network` it will default to using our archival bucket. (Not sure if it will be requester pays or not for archival - you'd probably know better)

Reply:
> Not sure if it will be requester pays or not for archival

No, not for the time-being